README.md

```diff
@@ -34,6 +39,31 @@ It is possible to send args to the script under test by adding -- after the load
 ./bin/load-runner.js -s ./scripts/dummy.js -c 5 -n 100 -r 10 -o -- -testarg1 testval1
 ```
 
+# Environment variables
+
+There are some environment variables generated for the test script that can be useful. You can use Node.js `process.env` object to access them.
+
+* `LR_FLOW_NUMBER` - number generated each run, value depends on percentages specified by `-f` parameter, see [Flow numbers](#flow-numbers)  
+* `LR_TOTAL_RUNS` - total number of runs, it's value of `-n` parameter
+* `LR_RUN_NUMBER` - current test run number
+* `LR_RAND` - randomly generated value for each run, float between 0 and 1, based on random generation with `--seed` parameter
+
+Look at [scripts/dummy_env_vars.js](scripts/dummy_env_vars.js) for example of using this.
+
+
+# Flow numbers
+
+Flow number functionality can be used inside test script to determine which branch of script to be executed. 
+It's specified by `-f` parameter and can be accessed by `LR_FLOW_NUMBER` environment variable in the script.
+
+For example this executes 5 times flow 0, 3 times flow 1, 2 times flow 2 from total of 10 runs.
+
+```
+./bin/load-runner.js -s ./scripts/dummy_flow_numbers.js -n 10 -o -f 50 30 20 --seed 123 
+```
+
+Flow numbers are randomly shuffled, but it's possible to use `--seed` parameter for repeatable result. 
```

Comment:
I'm not certain I fully understand the whole situation here, so feel free to ignore. I think a seed would make it impractical to run tests with different flows as even though they're repeatable, they don't provide deterministic flows.

For example, if I wanted to perform a load test where for each A there is two B's (ABBABBABB) I would have to run the tests a lot of times using different seeds until I determine the seed that results in this flow. I'd assume if I increased the total amount of tests being run by load runner the seed would then need to change also?

Similar to Dave's point, if I want to avoid clustering of `A`'s because they're heavier tasks that are run less frequently I would have to somehow determine the seed that results in `A`'s being adequately dispersed.

I think having an even distribution might be simpler for now. I hope that makes sense.